app/assets/javascripts/covid19/src/webapp/components/structures-table/Toolbar.tsx

```diff
@@ -33,6 +33,55 @@ export interface ToolbarProps {
 // Toolbar is called with empty object on initialization
 
 export const Toolbar: React.FC<ToolbarProps | {}> = props => {
+    const helpText = React.useMemo(() => {
+        return {
+            p: i18n.t(`Write in the searching box one (example: spike) or several
```

Comment:
Note that `yarn localize` does not add these literals to `en.pot`. That's because we have `i18next-scanner` errors (because of the ":", needs `nsSeparator: false`). `d2-i18n-extract` is to blame, as it exists with status 0 (OK) when in fact we have these errors.

I've added a fully typed i18t (from CPR), and added the `nsSeparator` (ideally, it should be a global configuration, but now possible with d2-i18n, we should open a PR sometime to Oslo)

Reply:
Thanks for your comments @tokland , I'm learning a lot!